src/code/bank0.asm

```diff
@@ -1992,22 +1992,23 @@ UseHookshot::
 ; Inputs:
 ;   a    inventory item
 label_1321::
```

Comment:
Nitpick: do you think we can rename this label `UseSword` then?

Reply:
I think the same, but this label already exists. So I investigation is needed, what it does beside the other function.
This function is used inside UseSword.
So I wanted to label stuff around it to get a better view.

Reply:
Seems suitable to me 👍